plugins/lookup/bitwarden.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# (c) 2022, Jonathan Lung <lungj@heresjono.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: bitwarden
+    author:
+      - Jonathan Lung (@lungj) <lungj@heresjono.com>
+    requirements:
+      - bw (command line utility)
+      - be logged into bitwarden
+    short_description: Retrieve secrets from bitwarden.
+    description:
+      - Retrieve secrets from bitwarden.
+    options:
+      _terms:
+        description: Key(s) to fetch values for from login info.
+        required: True
+      field:
+        description: Field to fetch; leave unset to fetch whole response.
+        default: None
+"""
+
+EXAMPLES = """
+- name: "Get 'password' from Bitwarden record named 'a_test'."
```

Comment:
Example `name:` should not have a period.
```suggestion
- name: "Get 'password' from Bitwarden record named 'a_test'"
```
(Also I would add a newline between the examples.)